forge-gui/res/cardsfolder/upcoming/wreck_hunter.txt

```diff
@@ -0,0 +1,12 @@
+Name:Wreck Hunter
+ManaCost:B B
+Types:Creature Human Artificer
+PT:2/2
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoosePlayer | TriggerDescription$ When CARDNAME enters the battlefield, choose target player. You create a tapped Powerstone token for each nonland card in that player's graveyard that was put there from the battlefield this turn. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
+SVar:TrigChoosePlayer:DB$ ChoosePlayer | ValidTgts$ Player | Choices$ ThisTargetedPlayer | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ X | TokenTapped$ True | TokenScript$ c_a_powerstone
```

Comment:
I think you can eliminate line 7 –
- [ ] move ValidTgts$ to this line
- [ ]  add `TokenOwner$ You` to this line
- [ ] change `Player.Chosen` in line 9 to TargetedPlayer

I think this should work, but if it doesn't you should add cleanup for chosenplayer